python-function/accelerator.yaml

```diff
@@ -79,6 +79,8 @@ engine:
           substitutions:
             - text: python-function
               with: "#artifactId"
+            - text: main
+              with: "#functionName"
```

Comment:
Out of curiosity, does this handle renaming the function definition in `func.py`?

Reply:
Ah, thanks for catching that!  I agree and have changed the default value to `main`

Reply:
yes, it does also rename the definition in func.py